### PR TITLE
add support for change_column

### PR DESCRIPTION
**Pull request by u007**
only sqlite not supported, requires dropping and adding new column for sqlite

### CODE REVIEW

**Comment by markbates** [comment]
I can't accept this unless SQLite is also supported, sorry. Look at the implementations of things like `drop_column` and `remove_column` to see how it's done.

**Comment by u007** [comment]
Hi Markbates,

i dont recommend running drop and recreate, as there might be existing data going on.
if they wish to do it, they have to do it manually by calling drop and and create column

**Comment by markbates** [comment]
Yes, that is why you have to transfer the data over, like we do for the other column types. See https://github.com/markbates/pop/blob/master/fizz/translators/sqlite.go for examples on how to do that.

**Comment by u007** [comment]
sqlite change_column support added

**Comment by markbates** [comment]
@u007 other than those few comments, this looks great!

I really appreciate the effort you've been putting into Pop. It's been really great, thank you so much.

**Comment by u007** [comment]
already did

**Comment by mkozjak** [comment]
What's the status with this issue?

**Comment by u007** [comment]
it should be good to merge, ive also added query.groupBy

**Comment by markbates** [comment]
@u007 I had asked for changes a while ago, can you please review them? Thanks.

**Comment by u007** [comment]
@markbates sorry , what did i missed?

**Comment by markbates** [comment]
@u007 there  are a bunch of comments listed above.

**Comment by u007** [comment]
im seing only these 2 comments, in which ive already followed and fixed, what am i missing?:
```
I can't accept this unless SQLite is also supported, sorry. Look at the implementations of things like drop_column and remove_column to see how it's done.
Yes, that is why you have to transfer the data over, like we do for the other column types. See https://github.com/markbates/pop/blob/master/fizz/translators/sqlite.go for examples on how to do that.
```

**Comment by markbates** [comment]
In the changes tab there are two inline comments.

**Comment by u007** [comment]
sorry , ive been exploring everywhere to find inline comment, where can i locate it?

**Comment by markbates** [comment]
@u007 my bad. Apparently I never hit the "request changes" button. I could see them, but you couldn't. Terrible UX on GitHub part.

File: fizz/translators/postgres.go [hunk 1]
```diff
@@ -120,17 +129,42 @@ func (p *Postgres) RenameIndex(t fizz.Table) (string, error) {
 	return fmt.Sprintf("ALTER INDEX \"%s\" RENAME TO \"%s\";", oi.Name, ni.Name), nil
 }
 
-func (p *Postgres) buildColumn(c fizz.Column) string {
-	s := fmt.Sprintf("\"%s\" %s", c.Name, p.colType(c))
-	if c.Options["null"] == nil {
-		s = fmt.Sprintf("%s NOT NULL", s)
+// buildtype: change / add
+func (p *Postgres) buildColumn(c fizz.Column, buildType string) string {
```
Comment (u007) [inline fizz/translators/postgres.go]: @markbates did u mean to split buildColumn into two sub function, one for add and another for change?

File: fizz/README.md [hunk 2]
```diff
@@ -56,6 +56,13 @@ add_column("table_name", "column_name", "string", {})
 
 See [above](#column-info) for more details on column types and options.
 
+## Alter a column
+
+Not supported in sqlite
```
Comment (u007) [inline fizz/README.md]: done

**Comment by markbates** [comment]
Merged! Thanks!

**Pull request closed without merging**